cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server.go

```diff
@@ -97,6 +103,29 @@ func NewServer(configGetter server.KubernetesConfigGetter) *Server {
 			}
 			return typedClient, dynamicClient, nil
 		},
+		actionConfigGetter: func(ctx context.Context, namespace string) (*action.Configuration, error) {
+			if configGetter == nil {
+				return nil, status.Errorf(codes.Internal, "configGetter arg required")
+			}
+			config, err := configGetter(ctx)
+			if err != nil {
+				return nil, status.Errorf(codes.FailedPrecondition, fmt.Sprintf("unable to get config : %v", err))
+			}
+
+			restClientGetter := agent.NewConfigFlagsFromCluster(namespace, config)
+			clientSet, err := kubernetes.NewForConfig(config)
+			if err != nil {
+				return nil, status.Errorf(codes.FailedPrecondition, fmt.Sprintf("unable to create kubernetes client : %v", err))
+			}
+			// TODO(mnelson): Update to allow different helm storage options.
```

Comment:
Currently is just either `configmap`, `secret` or  `sql (PostgreSQL)`, no? If we ever continue using Postgresql as the shared cache layer, we could also offer out-of-the-box support for it. Interesting

Reply:
> Currently is just either `configmap`, `secret` or `sql (PostgreSQL)`, no?

The current helm support defaults to secret but allows `configmap` as an option. I wasn't aware of the newer PostgresSQL support as a Helm storage driver, but we don't support this currently.

> If we ever continue using Postgresql as the shared cache layer, we could also offer out-of-the-box support for it. Interesting

Possibly, though it would have other implications too (eg. we use the postgresql more like a cache and don't worry about guarantees currently, or another, currently access to releases in secrets/configmaps is limited by RBAC, but access to releases in the DB would need to be manually checked per namespace like we do assets etc.)

Reply:
I don't know when Helm added support for Postgresql, but I have assumed it also was a reason to migrate to pg indeed.
Yep, Helm also mentions the permissions thing: https://helm.sh/docs/topics/permissions_sql_storage_backend/